tutorials/webpack/url-loader.md

```diff
@@ -0,0 +1,47 @@
+Webpack's `url-loader` is a nice choice to compress large files
+for storage. In the options property, you can specify the max `limit`
```

Comment:
"You can specify the max `limit` file size, in bytes, that Webpack will compress."